sites-enabled/20-www.freecodecamp.dev.conf

```
@@ -162,6 +162,20 @@ server {
     include snippets/common/proxy-params.conf;
   }
 
+  # reverse proxy news (italian) currently redirecting to english
+  location /italian/news {
+    if (-f /etc/nginx/maintenance/NEWS.txt) {
```

```suggestion
    if (-f /etc/nginx/maintenance/NEWS-ITA.txt) {
```
See previous discussion.